Makefile

```diff
@@ -189,7 +189,7 @@ ifndef HAS_GOX
 	($(GO) get $(PKG_GOX))
 endif
 ifndef HAS_GOLANGCI
-	(curl -sfL $(PKG_GOLANGCI_LINT_SCRIPT) | sh -s -- -b ${GOPATH}/bin v${PKG_GOLANGCI_LINT_VERSION})
+	(curl -sfL $(PKG_GOLANGCI_LINT_SCRIPT) | sh -s -- -b $$(go env GOPATH)/bin v${PKG_GOLANGCI_LINT_VERSION})
```

Comment:
Is there a reason for the double `$$`?
E.g. for me `go env GOPATH` returns `/home/me/go` and the additional `$` before that will break it :thinking:

Reply:
That is used for escaping the shell command `$(go env GOPATH)`. But, I think it would be better to define a Makefile variable `GOPATH = $(shell go env GOPATH)` and use it here...

Reply:
Yeah, sorry, I confused that with something different.
@cworth-gh, what do you think about @inercia's suggestion?